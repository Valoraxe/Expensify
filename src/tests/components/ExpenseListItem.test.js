import React from 'react'
import { shallow } from 'enzyme'
import ExpneseListItem from '../../components/ExpenseListItem'
import expenses from '../fixtures/expenses'

test("render expense list item", () => {
  const wrapper = shallow(<ExpneseListItem {...expenses[0]} />)
  expect(wrapper).toMatchSnapshot()
})
