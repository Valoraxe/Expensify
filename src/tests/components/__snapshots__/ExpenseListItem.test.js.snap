// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render expense list item 1`] = `
<Link
  className="list-item"
  replace={false}
  to="/edit/1"
>
  <div>
    <h3
      className="list-item_title"
    >
      Buddy
    </h3>
    <span
      className="list-item_subtitle"
    >
      01/01/1970
    </span>
  </div>
  <div>
    <h3
      className="list-item_data"
    >
      Â£
      3.00
    </h3>
  </div>
</Link>
`;
